CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [8.1.1]
+
+### ⚠️ Breaking changes
```

Comment:
Can you remove this since it's no longer a breaking change? Once that's fixed we can merge / release. Thanks!

Reply:
Done. Removed the breaking changes notice.